pkg/porter/cnab_test.go

```diff
@@ -190,12 +190,13 @@ func TestSharedOptions_LoadParameters(t *testing.T) {
 }
 
 func TestSharedOptions_CombineParameters(t *testing.T) {
-	ctx := context.Background()
+	c := portercontext.NewTestContext(t)
```

Comment:
same here

Reply:
I realized that the change I had made to this function (combineParameters) was because I _thought_ I needed access to the verbosity level. Since I really just needed the debug flag, I reverted the change.

I'll back this out and do a separate PR with the changes needed once we have a dedicated --debug flag on the porter install/upgrade/uninstall commands, separate from the debug flag that is being deprecated on the porter context.